src/mocks/index.js

```diff
@@ -7,6 +7,12 @@ import { AccommodationStatus } from "models/constants/AccomodationStatus";
 import { HostStatus } from "models/constants/HostStatus";
 import { GuestPriorityStatus } from "models/constants/GuestPriorityStatus";
 
+const languagesSpoken = ["pl", "ua", "en", "ger"];
```

Comment:
"ger" -> "de" :) Please use ISO 639-1 for reference. Please add a task in GitHub for some better implementation of this which allows for a complete list of locales.

Reply:
Also, there's `chance.locale();`